### DISCUSSION TITLE
Sort connections by active/inactive

### DISCUSSION THREAD

**Discussion by benisai** [wg-easy/wg-easy#108] [Ideas]
Hello Devs, this is my first wireguard gui I love it. I find myself scrolling to see which connections are active. 

Would it be possible to sort the active connections to the top? 
I have about 10-15 connections and I would like to see the active ones (with the red dot) be sorted at the top and the inactive at the bottom :) 
Would it be possible for you to code this?

**Comment by pheiduck** [comment]
@benisai I don't currently see that this feature is desired by several people. I would therefore close this. If it does come up for discussion again, it is certainly worth considering. :)
